recipes/cpr/all/conanfile.py

```diff
@@ -1,9 +1,13 @@
+from typing import no_type_check
```

Comment:
```suggestion
```

Because this will break python2 and there is no typing in conanfiles at all.

Reply:
Bleh, I think my editor added this in and I didn't catch it. Thanks.